docs/src/tutorials/anomaly_detection_unsupervised.ipynb

```diff
@@ -157,180 +124,19 @@
    "source": [
     "## Load data\n",
     "\n",
-    "We use Pandas to load the data into a `DataFrame` and perform tabular manipulation before importing it into a Temporian as an `EventSet`.\n",
-    "\n",
-    "We index the data by the `machine` column in Temporian. This was, we can apply Temporian transformations on each machine individually as well as across all machines."
+    "We use Pandas to load the data into a `DataFrame`.\n",
+    "Often, data requires cleaning before it can be used, and Pandas si great in this task.\n",
```

Comment:
si great in this task. -> is great for this task

Reply:
*Done